docs/source/index.rst

```diff
@@ -6,36 +6,56 @@
 Substra documentation
 =====================
 
-Substra is a framework offering distributed orchestration of machine learning tasks among partners while guaranteeing secure and trustless traceability of all operations.
+Substra is an open source federated learning framework. It provides a flexible python interface and a web app to run federated learning training at scale.
 
-- :doc:`Getting started <get_started/overview>`
+Substra main usage is a production one. It has already been deployed and used by hospitals and biotech companies. Yet Substra can also be used on a single machine on a virtually splitted dataset for two use cases:
+* debugging code before launching experiments on a real network
+* performing FL simulations
 
-  An overview and guide to install Connnect python library.
+Substra was created by `Owkin <https://owkin.com/>`_ and is now hosted by the `Linux Foundation for AI and Data <https://lfaidata.foundation/>`_.
 
-- :doc:`Documentation <documentation/concepts>`
+Interfaces
+^^^^^^^^^^
 
-  In depth documentation of Substra concepts and API reference.
 
-- :doc:`Examples <auto_examples/index>`
+Substra has four user interfaces:
+    * **Substra**: a low-level Python library (also called SDK). Substra is used to create datasets, algorithms and machine learning tasks on the platform.
+    * **Substrafl**: a high-level federated learning Python library based on Substra. Substrafl is used to run complex federated learning experiments at scale.
+    * a CLI - Command Line Interface
+    * a web interface
 
-  Examples of usage of Substra.
 
-- :doc:`Releases notes <additional/release>`
+Installation
+^^^^^^^^^^^^
 
-  Substra changelog and compatibility table.
+To install the two python libraries and the CLI run the following command:
 
-- :doc:`Operating a Substra network <operations/index>`
+.. code-block:: console
 
+    $ pip install substrafl
+
+Substra is a dependency of Substrafl, so it will be automatically installed.
+
+Links
+^^^^^
+
+Some quick links:
+
+* :ref:`MNIST federated learning example <substrafl_doc/examples/strategies_examples/plot_substrafl_fedavg:Substrafl FedAvg on MNIST dataset>`
```

Comment:
a `*` appears on the doc, there is a formatting problem here

Reply:
fixed